### PR TITLE
Changes to Matt's project as a result of a code review.

### PR DESCRIPTION
**Pull request by mmmayer**
Hi Matt -

We met during WWDC as part of the Community Week Mentorship Labs.  You invited me to take a look at this repository.  Sorry for taking a bit of time to review the code here.  I have submitted a pull request with my suggested changes.  Please feel free to reach out to me with questions.  Hope you have been having a tremendous Summer.

Regards, Michael
m.mayer6@gmail.com

### CODE REVIEW

File: EZ Tic Tac Toe/Models/GameState.swift [hunk 1]
```diff
@@ -15,23 +15,21 @@ class GameState: ObservableObject {
     static let emptyBoard: [[PositionValue]] = [[.empty, .empty, .empty], [.empty, .empty, .empty], [.empty, .empty, .empty]]
 
     ///Represents the state of the board as an enumeration
-    enum BoardState {
+    enum GamePhase {
```
Comment (MattGeimer) [inline EZ Tic Tac Toe/Models/GameState.swift:18]: Thoughts on `GamePhase` as opposed to `BoardState`? I agree that `BoardState` vs `GameState` was confusing.

File: EZ Tic Tac Toe/Models/Position.swift [hunk 2]
```diff
@@ -22,21 +22,22 @@ enum Position: String, CustomStringConvertible {  //3M: make code more Swift idi
...
+            case .topRight:     return (2,0)
+            // Middle row
+            case .middleLeft:   return (0,1)
+            case .center:       return (1,1)
+            case .middleRight:  return (2,1)
+            // Bottom row
+            case .bottomLeft:   return (0,2)
             case .bottomMiddle: return (1,2)
-            case .bottomRight: return (2,2)
+            case .bottomRight:  return (2,2)
```
Comment (MattGeimer) [inline EZ Tic Tac Toe/Models/Position.swift]: Do the comments/aligned returns help?

File: EZ Tic Tac Toe/SupportingViews/CreditView.swift [hunk 3]
```diff
@@ -16,22 +16,32 @@ struct CreditView: View {
...
+                    Text("Ben Newcomb")
+                    Text("Tyler Janczak")
+                }
+                VStack {
+                    Text("Additional Help")
+                        .font(.title)
+                    Text("Michael M. Mayer")
+                    Spacer()
+                }
+            })
```
Comment (MattGeimer) [inline EZ Tic Tac Toe/SupportingViews/CreditView.swift:37]: 😄

**Review by MattGeimer — COMMENTED** [review]
@mmmayer I looked through your changes! Thank you so much for taking a look and giving such great comments! I love some of the stuff you did like making things `CustomStringConvertable` where possible. I made a couple additional tweaks in response to some of the more open-ended comments. Could you take a look and let me know what you think?

File: EZ Tic Tac Toe/Models/GameState.swift (hunk 1 above)
Comment (mmmayer) [reply EZ Tic Tac Toe/Models/GameState.swift:18]: Naming is hard.  The name change is a big improvement.

File: EZ Tic Tac Toe/Models/Position.swift (hunk 2 above)
Comment (mmmayer) [reply EZ Tic Tac Toe/Models/Position.swift]: I didn't have any issue with the way it was, but having everything line up nicely is an improvement.

File: EZ Tic Tac Toe/SupportingViews/CreditView.swift (hunk 3 above)
Comment (mmmayer) [reply EZ Tic Tac Toe/SupportingViews/CreditView.swift:37]: You are very kind.  I enjoyed walking through your project and was just happy to help you put your best foot forward.  Please feel free to reach out if I can be of assistance in the future.  Good luck with your studies and internships.

**Comment by mmmayer** [comment]
My bad.  I should have run and updated the tests.  What and how to test is a whole other set of important skills.  Glad to see you developing solid professional habits in this area.

**Pull request merged**